### ISSUE TITLE
Using TX-NR545

### ISSUE THREAD

**Issue by kapps7** [gw-wiscon/homebridge-onkyo-avr#13]
Hey thanks for the plugin.

I am trying to use it with a TX-NR545. I have only changed the ip and the name. It finds but the switch on/off doesn't work. Can I do anything to make it work.

Thanks

**Comment by ToddGreenfield** [comment]
Hi - I am guessing that is a newer model from the NR535? If so, check out the very last comment on this page - https://www.npmjs.com/package/eiscp 
This package requires eiscp.  You could also see if you can successfully run the 3.js script in the example directory of eiscp (should be in homebridge-onkyo-avr/node_modules/eiscp/examples... at command prompt type "node 3.js" and see what the output shows for available commands).

**Comment by kapps7** [comment]
Thanks mate. I am trying to follow your instructions. I installed eiscp using this command easy_install onkyo-eiscp. I haven't goto to js yet.

I am simply trying onkyo --discover and nothing shows up. If I manually specify the host ip, it says non receivers found.
The device is active, shows up in bonjour and network scans.

**Comment by ToddGreenfield** [comment]
You need node installed and eiscp should install automatically when you install this plugin, however based on the comment (below) referenced in the link above I am guessing your model is too new and not supported. Not sure how to help you here- sorry.

"makes high-level commands unusable for new models which are not yet presented in the yaml file or for which no Onkyo documentation is available yet (as the time of this writing, for example, the latest V 1.26 .xls does not cover the NR535)"

**Comment by kapps7** [comment]
Thanks. So I put the dependency in package.json. Unfortunately no change in the outcome. This is the dump when I ran node 3.js

http://pastebin.com/g3YNnafK

**Comment by ToddGreenfield** [comment]
Looks like it should work. That dump looks fine. Are you able to use the iOS app like OnkyoRemote3 to control the receiver?

**Comment by kapps7** [comment]
Ok I got it working! I took the mains out of the amp for a few seconds. It seems that even though bonjour was showing, it actually wasn't connecting to the network.

Just a couple of follow ups.
The accessory is indicating off when the amp is on and vice versa: Here is the log:
setPowerState - actual mode, power state: 0, switching to OFF
[3/19/2017, 11:14:57 PM] [Unused] eventSystemPower - message: standby, new state false
[3/19/2017, 11:14:57 PM] [Unused] setPowerState - polling mode, ignore, state: false
[3/19/2017, 11:15:14 PM] [Unused] setPowerState - actual mode, power state: 1, switching to ON
[3/19/2017, 11:15:14 PM] [Unused] eventSystemPower - message: on, new state true

Also is it possible to set the volume.

Thanks very much!
